crates/proof-of-sql/src/sql/proof_exprs/inequality_expr_test.rs

```diff
@@ -477,28 +477,6 @@ fn we_can_compare_a_constant_column_of_zeros() {
     assert_eq!(res, expected_res);
 }
 
-#[test]
-fn the_sign_can_be_0_or_1_for_a_constant_column_of_zeros() {
-    let data = owned_table([bigint("a", [0_i64, 0, 0]), bigint("b", [1_i64, 2, 3])]);
-    let t = "sxt.t".parse().unwrap();
-    let accessor = OwnedTableTestAccessor::<InnerProductProof>::new_from_table(t, data, 0, ());
-    let mut ast = filter(
-        cols_result(t, &["b"], &accessor),
-        tab(t),
-        lte(column(t, "a", &accessor), const_bigint(0)),
-    );
-    if let DynProofPlan::Filter(filter) = &mut ast {
```

Comment:
Can you replace this with a `DenseFilter` rather than just removing it?

Reply:
Yup.